README.md

```
@@ -1,43 +1,78 @@
-* Fetching data from POCT devices
-* Report APIs
-* Language API
-* Notification service
-* CRM Reports
-* Appointment Scheduling
-
+# AMRIT - COMMON-PAI Service
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)  ![branch parameter](https://github.com/PSMRI/COMMON-PAI-API/actions/workflows/sast.yml/badge.svg)
+COMMON-PAI (Fast Healthcare Interoperability Resources) standard defines how healthcare information can be exchanged between different computer systems regardless of how it is stored in those systems. COMMON-PAI provides a means for representing and sharing information among clinicians and organizations in a standard way regardless of the ways local EHRs represent or store the data. COMMON-PAI combines the best features of previous standards into a common specification, while being flexible enough to meet needs of a wide variety of use cases within the healthcare ecosystem. Resources are the basis for all exchangeable COMMON-PAI content. Each resource includes a standard definition and human-readable descriptions about how to use the resource. Each resource also has a set of common and resource-specific metadata (attributes) to allow its use clearly and unambiguously. COMMON-PAI Resources can store and/or exchange many types of clinical and administrative data.
+In AMRIT, currently we have developed 9 resources out of 27 resources. Contributors are working on developing rest of the 18 resources which will make AMRIT to be compliant with ABDM guidelines. COMMON-PAI R4 is the latest version which we are migrating from HL7 V2.0 current version of AMRIT application.
+### Key APIs in COMMON-PAI service
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)